pkg/cosign/common.go

```diff
@@ -26,6 +26,14 @@ import (
 	"golang.org/x/term"
 )
 
+// confirmation is a global variable to store whether or not the user has provided
+// the --yes flag to skip all confirmation prompts
+var confirmation bool
```

Comment:
`confirmation` is IMO an ambiguous name.

How about `skipConfirmation` or similar?

Reply:
Done